### PR TITLE
Add PhD Thesis and ideal.II paper

### PR DESCRIPTION
**Pull request by jpthiele**

### CODE REVIEW

File: publications-2024.bib [hunk 1]
```diff
@@ -1251,6 +1251,26 @@ @Article{2024:talukdar.swaminathan.ea:stress-induced
...
+}
+
+@Misc{2024:thiele:idealii,
+  author  = {Thiele, Jan Philipp},
+  title   = {ideal.{II}: a Galerkin Space-Time Extension to the Finite Element Library deal.{II}},
+  year    = 2024,
+  url     = {https://arxiv.org/abs/2408.08840},
+  doi     = {10.48550/arXiv.2408.08840},
+  publisher = {arXiv},
+  preprint = {https://arxiv.org/abs/2207.04764},
```
Comment (marcfehling) [inline publications-2024.bib:1270]:
```suggestion
```
This belongs to the article below :)

**Review by marcfehling — CHANGES_REQUESTED** [review]

**Comment by bangerth** [comment]
Congratulations on defending successfully, @jpthiele !

File: publications-2024.bib (hunk 1 above)
Comment (jpthiele) [reply publications-2024.bib]: Ye olde Copy & Paste.. Good you spotted this!

**Comment by jpthiele** [comment]
> Congratulations on defending successfully, @jpthiele !

Thank you :)

**Review by marcfehling — APPROVED** [review]

**Pull request merged**